my-app/src/app/modules/courses-page/services/courses-handler.service.mock.ts

```diff
@@ -0,0 +1,25 @@
+import {ICourse} from "@shared/*";
+import {COURSE_DATA} from "@assets/mocks/course-data.mock";
+
+interface ICoursesHandlerService {
```

Comment:
You can put the interface into `my-app/src/app/modules/courses-page/services/courses-handler.service.ts` and make `CoursesHandlerService` also implement it

Reply:
Added, thank you